packages/blocksAntd/src/blocks/TextArea/TextArea.json

```diff
@@ -74,6 +74,20 @@
               "type": "string",
               "description": "Label title."
             },
+            "span": {
+              "type": "number",
+              "description": "Label inline span."
+            },
+            "inline": {
+              "type": "boolean",
+              "defaultValue": false,
+              "description": "Render input and label inline."
+            },
+            "disabled": {
+              "type": "boolean",
+              "default": false,
+              "description": "Disable the block if true."
```

Comment:
disabled label only not input, label.disabled etc should be same on all label fields